datafusion/physical-expr/src/expressions/case.rs

```diff
@@ -19,7 +19,8 @@ use std::{any::Any, sync::Arc};
 
 use crate::expressions::try_cast;
 use crate::PhysicalExpr;
-use arrow::array::{self, *};
+use arrow::array::*;
+use arrow::compute::kernels::zip::zip;
```

Comment:
Today I Learned: https://docs.rs/arrow/11.1.0/arrow/compute/kernels/zip/fn.zip.html ✅

Reply:
it certainly isn't the same as `zip` in Rust's iterators -- so I agree it is confusing